.gitignore

```diff
@@ -1,5 +1,4 @@
 *.py[cod]
-.*
```

Comment:
Revert this. I'm not sure why it's there but I suspect there is a good reason. I know why you removed it: you wanted to add the .github but you can do that with `git add -f` or you can add an exception `!/.github` in the .gitignore instead

Reply:
That may be because of some `.*` dirs/files added by IDEs. Okay, I have reverted it and added the `!/.github` line.

Reply:
I'm not sure. I don't use an IDE I use ViM. But it's not impossible that I did that for my colleagues back in the days. If you become maintainer feel free to remove it but remember there might be a reason it was there